spec/system/publishers/publishers_can_manage_self_disclosure_spec.rb

```diff
@@ -119,8 +119,8 @@
             )
 
             expect(all(".timeline-component__value").map { |x| x.text.split.first(6).join(" ") })
-              .to eq(["#{jobseeker.jobseeker_profile.personal_details.first_name} #{jobseeker.jobseeker_profile.personal_details.last_name} - #{Date.current.to_fs}",
-                      "#{publisher.given_name} #{publisher.family_name} - #{Date.current.to_fs}"])
+              .to eq(["#{jobseeker.jobseeker_profile.personal_details.first_name} #{jobseeker.jobseeker_profile.personal_details.last_name} -#{Date.current.to_fs}",
+                      "#{publisher.given_name} #{publisher.family_name} -#{Date.current.to_fs}"])
```

Comment:
Heads up, this was already fixed in the base branch during the latest post `main` merging fixes: https://github.com/DFE-Digital/teaching-vacancies/commit/bffc3903b7929bdd63364aca0dba8d8fd4184e95 

You can discard this/rebase and be conflict-free!